src/main/java/com/training/assignments/miniproject/EmployeeOperation.java

```diff
@@ -0,0 +1,21 @@
+package com.training.assignments.miniproject;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class EmployeeOperation {
+    static List<EmployeeManagement> employeeList = new ArrayList<>();
+
+    public void createEmployee(String name, String address, int salary, int id) {
+        employeeList.add(new EmployeeManagement(name, address, salary, id));
+    }
+
+    public List<EmployeeManagement> getEmployee() {
+        return employeeList;
+    }
+
+    public List<EmployeeManagement> getDeletedEmployee(int index, List<EmployeeManagement> employee) {
```

Comment:
make this deleteEmployee with void.
This should just do the task of deleting the employee.
Why are we giving the employeeList here when we already have a static list defined globally?